volumes/vol1/03_prelims/22_sets/84_exercises_finitemap.tex

```diff
@@ -48,6 +48,8 @@
 For the data to be well-formed it is necessary that for each element in the domain, there is exactly one row for that element.
 If that is not the case, throw an exception \classname{InvalidFormat}.
 
+\todo{use \codeboilerplate to be consistent?}
```

Comment:
This will not work - codeboilerplate expects an argument. Use `\str{coderboilerplate}`.